addons/website_sale_collect_wishlist/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Product Availability Notifications',
```

Comment:
```suggestion
    'name': "Product Availability Notifications",
```

Reply:
Thank you for the suggestion! I noticed that Odoo’s codebase mainly uses single quotes in manifest files (923 times vs. 253 for double quotes). Just curious, is there a reason you’d prefer double quotes here?

Reply:
We add double quotes for UX strings which will be visible for users and single for technical strings
You can go through our knowledge article [coding Guidelines of sales](https://www.odoo.com/odoo/knowledge/8790) for more rationale :wink:

Reply:
This explains well. Thanks for this information! :smile: